packages/tooling/fast-tooling-react/src/form/templates/const-value.props.ts

```diff
@@ -1,3 +1,5 @@
+import { FormStrings } from "@microsoft/fast-tooling-react/src/form/form.props";
```

Comment:
This import should reference the relative filepath, eg. `"../form.props";`

Reply:
I have replaced all instances with a relative path.